<?xml version="1.0"?>
<launch>
  <arg name = "module_ids" default=""/>
  <arg name = "real_machine" default="true"/>
  <node pkg="ninja" type="disassembly_motion.py" name="" output="screen" >
    <param name = "module_ids" value="$(arg module_ids)"/>
    <param name = "real_machine" value="$(arg real_machine)"/>
  </node>
</launch>
